src/go.mod

```diff
@@ -8,8 +8,8 @@ require (
 	github.com/go-logr/logr v0.4.0
 	github.com/google/go-cmp v0.5.5
 	github.com/imdario/mergo v0.3.10 // indirect
-	github.com/opslevel/opslevel-go v0.4.2
-	github.com/rocktavious/autopilot v0.1.5 // indirect
```

Comment:
nit: you lost your comment here

Reply:
ya `go mod tidy` did that so i guess its direct dependency now that we are tests in this repo